wordpress-mobile/WordPress-iOS
# Fix mime type sent to server when uploading video media.

Fixes #5326 

To test:
 - On a self hosted site
 - Create a new post and add a device video to the post.
 - Save the post
 - Check the media type on the web server and see if it's the correct one.

Needs review: @aerych
